packages/@aws-cdk/aws-s3/test/bucket.test.ts

```diff
@@ -1175,165 +1101,63 @@ describe('bucket', () => {
-                        'Fn::Join': [
-                          '',
-                          [
-                            'arn:',
-                            {
-                              'Ref': 'AWS::Partition',
-                            },
-                            ':iam::',
-                            {
-                              'Ref': 'AWS::AccountId',
-                            },
-                            ':root',
-                          ],
-                        ],
-                      },
-                    },
-                    'Resource': '*',
-                  },
-                  {
-                    'Action': [
-                      'kms:Encrypt',
-                      'kms:ReEncrypt*',
-                      'kms:GenerateDataKey*',
-                      'kms:Decrypt',
-                    ],
-                    'Effect': 'Allow',
-                    'Principal': {
-                      'AWS': {
-                        'Fn::GetAtt': [
-                          'MyUserDC45028B',
-                          'Arn',
-                        ],
-                      },
-                    },
-                    'Resource': '*',
-                  },
-                ],
-                'Version': '2012-10-17',
-              },
-              'Description': 'Created by Default/MyBucket',
-            },
-            'UpdateReplacePolicy': 'Retain',
-            'DeletionPolicy': 'Retain',
-          },
-          'MyBucketF68F3FF0': {
-            'Type': 'AWS::S3::Bucket',
-            'Properties': {
-              'BucketEncryption': {
-                'ServerSideEncryptionConfiguration': [
-                  {
-                    'ServerSideEncryptionByDefault': {
-                      'KMSMasterKeyID': {
-                        'Fn::GetAtt': [
-                          'MyBucketKeyC17130CF',
-                          'Arn',
-                        ],
-                      },
-                      'SSEAlgorithm': 'aws:kms',
-                    },
-                  },
-                ],
-              },
-            },
-            'UpdateReplacePolicy': 'Retain',
-            'DeletionPolicy': 'Retain',
-          },
-          'MyUserDC45028B': {
-            'Type': 'AWS::IAM::User',
-          },
-          'MyUserDefaultPolicy7B897426': {
-            'Type': 'AWS::IAM::Policy',
-            'Properties': {
-              'PolicyDocument': {
-                'Statement': [
-                  {
-                    'Action': [
-                      's3:DeleteObject*',
-                      's3:PutObject*',
-                      's3:Abort*',
-                    ],
-                    'Effect': 'Allow',
-                    'Resource': [
+      expect(stack).toHaveResource('AWS::IAM::Policy', {
+        'PolicyDocument': {
+          'Statement': [
+            {
+              'Action': [
+                's3:DeleteObject*',
+                's3:PutObject*',
```

Comment:
The test is 'does *not* grant PutObjectAcl', but the Statement includes 's3.PutObject*'. I see that's what it was before though; is this test set up wrong?

Reply:
You're mis-reading the diff. The test is 'with KMS key has appropriate permissions for multipart uploads'